.goreleaser.yml

```diff
@@ -12,6 +12,12 @@ release:
     - glob: ./manifest/charts/*
     - glob: ./manifest/yamls/*
 
+variables:
+  fetcherDescription: Fetcher is a lightweight component used by environment and builder pods. Fetcher helps in fetch and upload of source/deployment packages and specializing environments.
+  builderDescription: The builder assists in building the fission function source code for deployment.
+  reporterDescription: The reporter gathers information that assists in improving fission.
+  preUpgradeChecksDescription: Preupgradechecks ensures that Fission is ready for the targeted version upgrade by performing checks beforehand.
+  fissionBundleDescription: fission-bundle is a component which is a single binary for all components. Most server side components running on server side are fission-bundle binary wrapped in container and used with different arguments.
```

Comment:
fission-bundle is a single binary containing most of the fission components, such as executor, router, and builder-manager. Each specific component is run by passing specific arguments.